docs/standard/parallel-programming/data-parallelism-task-parallel-library.md

```diff
@@ -40,7 +40,7 @@ ms.workload:
   
  Both the <xref:System.Threading.Tasks.Parallel.For%2A?displayProperty=nameWithType> and <xref:System.Threading.Tasks.Parallel.ForEach%2A?displayProperty=nameWithType> methods have several overloads that let you stop or break loop execution, monitor the state of the loop on other threads, maintain thread-local state, finalize thread-local objects, control the degree of concurrency, and so on. The helper types that enable this functionality include <xref:System.Threading.Tasks.ParallelLoopState>, <xref:System.Threading.Tasks.ParallelOptions>, <xref:System.Threading.Tasks.ParallelLoopResult>, <xref:System.Threading.CancellationToken>, and <xref:System.Threading.CancellationTokenSource>.  
   
- For more information, see [Patterns of Parallel Programming](http://go.microsoft.com/fwlink/p/?LinkId=265491).  
+ For more information, see [Patterns of Parallel Programming (download in PDF format)](https://download.microsoft.com/download/3/4/D/34D13993-2132-4E04-AE48-53D3150057BD/Patterns_of_Parallel_Programming_CSharp.pdf).  
```

Comment:
perhaps it should go to the same as the other topics?
https://www.microsoft.com/download/details.aspx?id=19222

Reply:
@mairaw that's true, fixed.